heron/tracker/src/python/version.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
problem is that implementation of argument parser of each CLI tool is different (because they are written by different people). You cannot reuse this ``version.py`` for `heron` and `heron-explorer`.